examples/ble_scanning.rs

```diff
@@ -15,7 +15,8 @@ struct LedCommand {
     pub st: bool,
 }
 
-async fn main() -> TockResult<()> {
+#[libtock_main]
+async fn main() -> libtock::result::TockResult<()> {
```

Comment:
Why did you stop `use`ing `libtock::result::TockResult`?

Reply:
Fixed.